### PR TITLE
Fix #277577: Name note input shortcuts the same as in menu

### PR DESCRIPTION
**Pull request by neGjodsbol**
Added texts in shortcut.cpp to be used in Preferences/shortcuts.

### CODE REVIEW

**Comment by neGjodsbol** [comment]
I have been assigned to this. Is there anything I have to do now?

**Comment by neGjodsbol** [comment]
This is still being requested. Any chance to have it accepted?

File: mscore/shortcut.cpp [hunk 1]
```diff
@@ -237,8 +237,8 @@ Shortcut Shortcut::_sc[] = {
          STATE_NORMAL | STATE_NOTE_ENTRY,
          "note-input",
          QT_TRANSLATE_NOOP("action","Note Input"),
+         QT_TRANSLATE_NOOP("action","Note input: Toggle Entry/Normal mode"),
          QT_TRANSLATE_NOOP("action","Note input"),
```
Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: this line doesn't add any value. Better remove it

File: mscore/shortcut.cpp [hunk 2]
```diff
@@ -237,8 +237,8 @@ Shortcut Shortcut::_sc[] = {
...
-         0,
```
Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: and leave this in then

**Comment by Jojo-Schmitz** [comment]
This indeed should get merged sooner rather than later

**Comment by neGjodsbol** [comment]
Do I need to take any action in this?
- Rebase, make change, force push

**Comment by Jojo-Schmitz** [comment]
yes

**Comment by Jojo-Schmitz** [comment]
Still needs a rebase. And as it is set to be a 3.3.1 milestone, sooner rather than later

**Comment by dmitrio95** [comment]
This PR adds new strings for translations, so better have it in 3.4 instead.

**Comment by Jojo-Schmitz** [comment]
But even then and there  a rebase is needed

**Comment by Jojo-Schmitz** [comment]
@neGjodsbol can you please rebase?

**Comment by Jojo-Schmitz** [comment]
Rebase is still needed

File: mscore/shortcut.cpp (hunk 1 above)
Comment (dmitrio95) [reply mscore/shortcut.cpp]: Do you mean exactly this "Note input" entry or all these third entries in shortcuts list?

Comment (Jojo-Schmitz) [reply mscore/shortcut.cpp]: Just this (I think)

Comment (Jojo-Schmitz) [reply mscore/shortcut.cpp]: It is a tooltip IIRC, which here, as just repeating information, isn't needed at all

**Comment by anatoly-os** [comment]
See rebased version of this PR: https://github.com/musescore/MuseScore/pull/5906

File: mscore/shortcut.cpp [hunk 3]
```diff
@@ -303,8 +303,8 @@ Shortcut Shortcut::_sc[] = {
          STATE_NORMAL | STATE_NOTE_ENTRY,
          "note-input-timewise",
          QT_TRANSLATE_NOOP("action","Timewise"),
+         QT_TRANSLATE_NOOP("action","Note input: Enter Timewise mode"),
```
Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp:306]: Insert mode?

Comment (Jojo-Schmitz) [reply mscore/shortcut.cpp:306]: We got rid of the term 'timewise' quite a while ago

**Pull request closed without merging**